hypershift-operator/controllers/nodepool/nodepool_controller_test.go

```diff
@@ -878,6 +929,9 @@ kind: Config`)},
 			}
 			g.Expect(missingConfigs).To(Equal(tc.missingConfigs))
 			g.Expect(err).ToNot(HaveOccurred())
+			if diff := cmp.Diff(mirroredConfigs, tc.mirroredConfigs); diff != "" {
```

Comment:
Please use cmpopts to explicitly not check resource version, as opposed to hard-coding it here to whatever controller-runtime fake is doing. We do not have a meaningful assertion over RV.

Reply:
Thanks, I wasn't aware of that. Will do

Reply:
Done